modules/default/weatherforecast/weatherforecast.js

```diff
@@ -339,7 +339,13 @@ Module.register("weatherforecast", {
 	 *
 	 * argument data object - Weather information received form openweather.org.
 	 */
-	processWeather: function (data) {
+	processWeather: function (data, momenttz) {
+		let mom;
+		if (momenttz === null) {
+			mom = moment;
+		} else {
+			mom = momenttz;
+		}
```

Comment:
Could be shorter (and still readable) like this 
let mom = !momenttz ? moment : mom;

So we need this for the tests?

Reply:
In that case I'd prefer:
```
let mom = momenttz ? mom : moment; // Exception last.
```

I'm also not a big fan of abbreviations in variable names. Looking at the above code, I've got no idea what `mom` is.

Reply:
short if added